### PR TITLE
apx: update page; apx-{pkgmanagers,stacks,subsystems}: add page

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

**Disclosure**: I am a maintainer at Vanilla OS. The future version of docs can be found [here](https://github.com/Vanilla-OS/documentation/blob/main/articles/en/apx-manpage.md)

### CODE REVIEW

File: pages/linux/apx-stacks.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`apx stacks update {{name}}`
+
+- List all available stack configurations:
+
+`apx stacks list`
+
+- Remove a specified stack configuration:
+
+`apx stacks rm --name {{string}}`
```
Comment (acuteenvy) [inline pages/linux/apx-stacks.md:21]:
```suggestion
`apx stacks rm --name {{name}}`
```

File: pages/linux/apx-stacks.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Import a stack configuration:
+
+`apx stacks import --input {{path/to/stack.yml}}`
+
+- Export a stack configuration (Note: the output flag is optional):
```
Comment (acuteenvy) [inline pages/linux/apx-stacks.md:27]: If the output flag is optional, what is the default? I think it should be added here.
```suggestion
- Export the stack configuration (Note: the output flag is optional):
```

File: pages/linux/apx-stacks.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`apx stacks export --name {{string}} --output {{path/to/output}}`
```
Comment (acuteenvy) [inline pages/linux/apx-stacks.md:29]:
```suggestion
`apx stacks export --name {{name}} --output {{path/to/output_file}}`
```

File: pages/linux/apx-subsystems.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Interactively create a new subsystem:
+
+`apx subsystems new`
+
+- List all available subsystems:
+
+`apx subsystems list`
+
+- Reset a specific subsystem to it's initial state:
```
Comment (acuteenvy) [inline pages/linux/apx-subsystems.md:15]:
```suggestion
- Reset a specific subsystem to its initial state:
```

File: pages/linux/apx-subsystems.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`apx subsystems reset --name {{string}} --force`
+
+- Remove a specific subsystem:
+
+`apx subsystems rm --name {{string}}`
+
+- [f]orce remove a specific subsystem:
+
+`apx subsystems rm --name {{string}} --force`
```
Comment (acuteenvy) [inline pages/linux/apx-subsystems.md:29]:
```suggestion
`apx subsystems reset --name {{name}}`

- Force reset a specific subsystem:

`apx subsystems reset --name {{name}} --force`

- Remove a specific subsystem:

`apx subsystems rm --name {{name}}`

- Force remove a specific subsystem:

`apx subsystems rm --name {{name}} --force`
```

File: pages/linux/apx-pkgmanagers.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`apx pkgmanagers create`
+
+- List all available package manager confirgurations:
+
+`apx pkgmanagers list`
+
+- Remove a package manager configuration:
+
+`apx pkgmanagers rm --name {{string}}`
```
Comment (acuteenvy) [inline pages/linux/apx-pkgmanagers.md:17]:
```suggestion
`apx pkgmanagers rm --name {{name}}`
```

File: pages/linux/apx-pkgmanagers.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# apx pkgmanagers
+
+> Manage package managers in `apx`.
+> Note: User created package manager configurations are stored in `~/.local/share/apx/pkgmanagers`.
```
Comment (acuteenvy) [inline pages/linux/apx-pkgmanagers.md:4]:
```suggestion
> Note: user-created package manager configurations are stored in `~/.local/share/apx/pkgmanagers`.
```

File: pages/linux/apx-stacks.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# apx stacks
+
+> Manage stacks in `apx`.
+> Note: User created stack configurations are stored in `~/.local/share/apx/stacks`.
```
Comment (acuteenvy) [inline pages/linux/apx-stacks.md:4]:
```suggestion
> Note: user-created stack configurations are stored in `~/.local/share/apx/stacks`.
```

File: pages/linux/apx-stacks.md (hunk 2 above)
Comment (kbdharun) [reply pages/linux/apx-stacks.md:27]: The file is outputted to your current directory by default, you can specify a different one using the output flag.

**Review by kbdharun — COMMENTED** [review]
@acuteenvy Thanks for your suggestions, I have applied some of your suggestions.

Note: We don't have these things on the style guide yet as we decide these things on a case-by-case basis. There was an effort by a previous maintainer to formalize it last year but the person has since left the project and started a different one (See our README similar projects you will see CLIP listed first) with extended syntax.

We use mnemonics (i.e. Display [h]elp; when the command supports passing `-h` as a flag for it) interchangeably in descriptions (even at the start) to highlight short options of commands as we prefer long options in examples. We have a lot of pages using mnemonics like this so I wouldn't suggest removing it here.

Regarding the placeholder description, we generally go with what the project uses/what is recommended in docs (In this case, I am the person who worked on the doc). Also, it is redundant to repeat the long option with the placeholder (last year in the unmerged draft we planned to use `{{string}}` as the placeholder, which is already used by some pages). If you still want me to convert it to `{{name}}` feel free to let me know.

File: pages/linux/apx-stacks.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+- Remove a specified stack configuration:
+
+`apx stacks rm --name {{string}}`
+
+- Import a stack configuration:
+
+`apx stacks import --input {{path/to/stack.yml}}`
+
+- Export the stack configuration (Note: the output flag is optional):
```
Comment (acuteenvy) [inline pages/linux/apx-stacks.md:27]:
```suggestion
- Export the stack configuration (Note: the output flag is optional - current working directory by default):
```

**Review by acuteenvy — APPROVED** [review]
I thought the mnemonics were only used if there are no long options. I haven't seen a lot of pages using them. Should all pages have short options marked like that?

**Comment by kbdharun** [comment]
> I thought the mnemonics were only used if there are no long options. I haven't seen a lot of pages using them. Should all pages have short options marked like that?

Yep, most of our very old pages do include it for instance the `tar` page that we use in our [banner and carousel](https://github.com/tldr-pages/tldr?tab=readme-ov-file#what-is-tldr-pages)

File: pages/linux/apx-stacks.md (hunk 9 above)
Comment (kbdharun) [reply pages/linux/apx-stacks.md]: Updated the wording a bit and applied the suggestion.

**Comment by kbdharun** [comment]
Found the old issue where we were discussing it https://github.com/tldr-pages/tldr/issues/8304, I will try to reopen the discussion soon.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**